src/components/film/film-header/film-header.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {FilmHeader} from 'src/components/film/film-header/film-header.js';
+import {shallow} from 'enzyme';
+
+const {title, rating} = {
```

Comment:
`const title = ...;
const rating = ...;` will be better

Reply:
ok